File: src/test/java/com/raychenon/leetcode/AddBinaryTest.java
package com.raychenon.leetcode;

import org.junit.BeforeClass;
import org.junit.Test;

import static junit.framework.TestCase.assertEquals;

/**
 * User: raychenon
 * Date: 3/3/2020
 * https://leetcode.com/problems/add-binary
 */
public class AddBinaryTest {
    
    private static AddBinary addBinary;

    @BeforeClass
    public static void setUp() {
        addBinary = new AddBinary();
    }

    @Test
    public void test_100() {
        assertEquals("100", addBinary.addBinary("11", "1"));
    }

    @Test
    public void test_10101() {
        assertEquals("10101", addBinary.addBinary("1010", "1011"));
    }

}

File: src/test/kotlin/com/raychenon/kotlin/leetcode/AddBinaryTest.kt
package com.raychenon.kotlin.leetcode

import junit.framework.TestCase
import org.junit.Test

/**
 * User: raychenon
 * Date: 4/3/2020
 * https://leetcode.com/problems/add-binary/
 */
class AddBinaryTest {

    @Test
    fun test_100() {
        TestCase.assertEquals("100", AddBinary.addBinary("11", "1"))
    }

    @Test
    fun test_10101() {
        TestCase.assertEquals("10101", AddBinary.addBinary("1010", "1011"))
    }
}